src/gui/gui.cpp

```diff
@@ -62,10 +88,15 @@ void init()
 	for(const config& g : guis_cfg.child_range("gui")) {
 		const std::string id = g["id"];
 
-		auto iter = guis.emplace(id, gui_definition(g)).first;
+		auto [iter, is_unique] = guis.emplace(id, gui_definition(g));
 
-		if(id == "default") {
-			default_gui = iter;
+		if (!is_unique) {
+			ERR_GUI_P << "GUI Theme ID '" << id << "' already exists.";
+			current_gui = default_gui;
```

Comment:
In case there are two themes with same ID, it's supposed to use default theme instead for the second one. In other words, the first one takes precedence.

Reply:
I don't get it. What does "use default theme instead for the second one" mean? The duplicate is not inserted at all. Also this loop isn't for selecting a theme is it? That's `switch_theme` below.

Reply:
> The duplicate is not inserted at all.

Didn't notice that. Will removing `current_gui = default_gui;` be enough?

Reply:
Yes, that is the part that makes no sense to me.

Reply:
Removed.